eng/versioning/external_dependencies.txt

```diff
@@ -16,7 +16,6 @@ com.microsoft.azure:azure-client-authentication;1.6.15
 com.microsoft.azure:azure-client-runtime;1.6.15
 com.microsoft.azure:azure-core;0.9.8
 com.microsoft.azure:azure-keyvault-cryptography;1.2.2
-com.microsoft.azure:azure-mgmt-keyvault;1.11.1
```

Comment:
This was a duplicate entry (both were the same). Since this is a test dependency I removed this one and left the one there. Yes, the verification script found this..